homeassistant/components/blebox/sensor.py

```diff
@@ -1,15 +1,46 @@
 """BleBox sensor entities."""
-from homeassistant.components.sensor import SensorDeviceClass, SensorEntity
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import TEMP_CELSIUS
+from homeassistant.const import CONCENTRATION_MICROGRAMS_PER_CUBIC_METER, TEMP_CELSIUS
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from . import BleBoxEntity, create_blebox_entities
 
-BLEBOX_TO_UNIT_MAP = {"celsius": TEMP_CELSIUS}
 
-BLEBOX_TO_SENSOR_DEVICE_CLASS = {"temperature": SensorDeviceClass.TEMPERATURE}
+@dataclass
+class BleboxSensorEntityDescription(SensorEntityDescription):
+    """Class describing Blebox sensor entities."""
+
+
+SENSOR_TYPES = (
+    BleboxSensorEntityDescription(
+        key="pm1",
+        device_class=SensorDeviceClass.PM1,
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    ),
+    BleboxSensorEntityDescription(
+        key="pm2_5",
+        device_class=SensorDeviceClass.PM25,
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    ),
+    BleboxSensorEntityDescription(
+        key="pm10",
+        device_class=SensorDeviceClass.PM10,
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    ),
+    BleboxSensorEntityDescription(
+        key="temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        native_unit_of_measurement=TEMP_CELSIUS,
+    ),
+)
```

Comment:
Maybe this can be in this PR, but then please update the description explaining that the air quality entity has been replaced with sensors.

Reply:
Updated.